ovos_core/intent_services/__init__.py

```diff
@@ -68,22 +58,23 @@ def __init__(self, bus):
 
         # TODO - replace with plugins
         self.adapt_service = AdaptService()
-        if PadaciosoService is not PadatiousService:
+        try:
+            from ovos_core.intent_services.padatious_service import PadatiousService
             self.padatious_service = PadatiousService(bus, config['padatious'])
-        else:
-            LOG.error(f'Failed to create padatious handlers, padatious not installed')
+        except ImportError:
+            LOG.error(f'Failed to create padatious intent handlers, padatious not installed')
             self.padatious_service = None
```

Comment:
This only handles `ImportError` exceptions, it should also capture and log other exceptions

Reply:
we only want to handle import error, due to padatious being optional, its the only expected exception to happen

any other error should crash things instead of pretending all is good no?